src/components/Tabs.js

```diff
@@ -26,9 +26,18 @@ class Tabs extends React.Component {
 
   state = {
     selectedTab: this.props.selectedTab || 0,
-    showMenu: false
+    showMenu: false,
+    singleTab: false
   };
 
+  componentDidMount () {
+    if (this.props.tabs.length === 1) {
```

Comment:
What made you want to use state here instead of relying on the prop.tabs.length check on its own?

Reply:
That's a good point. I thought I was going to need it in more places, but I'm only using it once. I'll make the change.